src/App.tsx

```diff
@@ -91,6 +97,14 @@ function App() {
         setSettingState( (settingState + 1 ) % 4 );
     }
 
+    function shareSessionStateChangeHandler() {
+        // 0: Closed
+        // 1: Opening
+        // 2: Open
+        // 3: Closing
+        setShareSessionState( (shareSessionState + 1 ) % 4 );
```

Comment:
Can you make a new enum in a file called something like `Types/UIStateTypes.tsx` and use this to manage state instead of using this.

Reply:
I added a UIStateTypes enum for Settings and ShareSession states in the latest commit.